src/primitives.jl

```diff
@@ -206,22 +206,6 @@ unwrapped `Nullable` entries.
 dropnull!(X::NullableVector) = deleteat!(X, find(X.isnull)).values # -> Vector
 
 """
-    anynull(X)
-
-Returns whether or not any entries of `X` are null.
-"""
-anynull(X::Any) = any(_isnull, X)           # -> Bool
-anynull(X::NullableArray) = any(X.isnull)   # -> Bool
```

Comment:
Oh, right, my bad. That's what @ararslan said in https://github.com/JuliaStats/NullableArrays.jl/pull/189#issuecomment-294309229. I misunderstood what your proposed change to Base would accomplish, although I probably should have realized that the proposed change would only affect Base types. I'll fix this, thanks for clarifying!